content/agent/faq/upgrade-to-agent-v6.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Upgrade to Datadog Agent v6
+kind: faq
+---
+
+## Amazon Linux
+
+A script is available to automatically install or upgrade to the new Agent. It sets up the package repositories and installs the Agent package for you. When upgrading, the import tool also searches for an existing `datadog.conf` from a prior version, and converts Agent and Check configurations according to the new v6 format.
+
+### One-step install
+
+The Agent v6 installer can automatically convert v5 configurations during the upgrade:
+```shell
+DD_UPGRADE=true bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
+```
+
+**Note:** The import process won't automatically move **custom** Agent checks. This is by design as we cannot guarantee full backwards compatibility out of the box.
+
+#### Fresh install
```

Comment:
Let's remove all the `Fresh install` sections, these are documented on https://app.datadoghq.com/account/settings#agent